Supervision Work/src/uk/ac/cam/ap886/oopjava/supervisionProjects/Hangman/GUIMainWindow.java

```diff
@@ -0,0 +1,203 @@
+package uk.ac.cam.ap886.oopjava.supervisionProjects.Hangman;
+
+import java.awt.ComponentOrientation;
+import java.awt.Dimension;
+import java.awt.EventQueue;
+import java.awt.Font;
+
+import javax.swing.BoxLayout;
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JOptionPane;
+import javax.swing.JPanel;
+import javax.swing.JSeparator;
+
+public class GUIMainWindow implements UserInterface{
+	Game game;
+	boolean firstRun=true;
+	
+	JLabel lblName;
+	JLabel lblGamesWon;
+	JLabel lblLettersTried;
+	JLabel openedWord;
+	JLabel lblStatus;
+	GUIGamePanel hangmanPanel;
+
+	public GUIMainWindow (Game newGame){
+		game = newGame;
+		initialize();
+	}
+
+	private JFrame frame;
+	private JSeparator separator;
+
+	/**
+	 * Launch the application.
+	 */
+	public static void main(String[] args) {
```

Comment:
This is a test method for starting the game panel without the game itself. It's for testing purposes only.